examples/haproxy/lego-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kube-lego
+  namespace: kube-lego
+spec:
+  selector:
+    matchLabels:
+      app: kube-lego
+  template:
+    metadata:
+      labels:
+        app: kube-lego
+    spec:
+      containers:
+      - name: kube-lego
+        image: jetstack/kube-lego:0.1.6
```

Comment:
For now I think we should replace `0.1.6` with `canary`, and perhaps add a comment explaining that HAProxy support isn't in a cut release yet.

Before we actually cut a release, we usually like to see success/become familiar with bug reports from users of the :canary tag using the new feature. After a few weeks I'll then cut a 0.1.6 release 😄

Reply:
> For now I think we should replace 0.1.6 with canary, and perhaps add a comment explaining that HAProxy support isn't in a cut release yet.

Makes sense. Just changed the tag.